database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -789,6 +789,14 @@ IMMUTABLE;
 
 -- Add columns to "review" now that these functions are defined:
 ALTER TABLE public.review
+    DROP COLUMN IF EXISTS application_id CASCADE,
+    DROP COLUMN IF EXISTS reviewer_id CASCADE,
+    DROP COLUMN IF EXISTS level_number CASCADE,
+    DROP COLUMN IF EXISTS stage_number CASCADE,
+    DROP COLUMN IF EXISTS time_stage_created CASCADE,
+    DROP COLUMN IF EXISTS is_last_level CASCADE,
+    DROP COLUMN IF EXISTS is_last_stage CASCADE,
+    DROP COLUMN IF EXISTS is_final_decision CASCADE,
```

Comment:
So weird how this is required here, but not for the "template_id" generated column in `review_assignment` 🤷‍♂️